dangerzone/isolation_provider/container.py

```diff
@@ -217,6 +218,37 @@ def _convert(
         document: Document,
         ocr_lang: Optional[str],
         stdout_callback: Optional[Callable] = None,
+    ) -> bool:
+        # This path needs to be permanent, and may already exist from previous runs.
+        os.makedirs(self.cache_dir, exist_ok=True)
+
+        # Create a temporary directory inside the cache directory for this run. Then,
+        # create some subdirectories for the various stages of the file conversion:
+        #
+        # * pixel: Where the RGB data will be stored
+        # * safe: Where the final PDF file will be stored
+        with tempfile.TemporaryDirectory(dir=self.cache_dir) as t:
```

Comment:
Yeah, you're right. The proper place for these temp files certainly wasn't the config directory, and it probably isn't the cache dir. I didn't want to stir things up, but since you're pointing it out as well, I'll send a quick fix.